universal-application-tool-0.0.1/app/controllers/admin/QuestionController.java

```diff
@@ -69,29 +70,40 @@ public QuestionController(
                   for (QuestionServiceError e : result.getErrors()) {
                     messageJoiner.add(e.message());
                   }
-                  errorMessage = messageJoiner.toString();
+                  String errorMessage = messageJoiner.toString();
+                  return withData(
+                      withMessage(redirect(routes.QuestionController.newOne()), errorMessage),
+                      questionForm.getData());
```

Comment:
Suggestion (blocking): When creation fails due validation failures, re-render the form instead of redirecting. You won't need to store the form data or validation messages in the flash this way.

Generally speaking, the flash should be used for simple string messages.

Reply:
Done.